src/pages/Home/Home.js

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
+import React from "react";
 
-import { Button } from 'components/button/Button';
+import "./Home.scss";
+import { ProfileTop, ProfileBottom, ProfileCard } from "../../components/Profile/Profile";
 
-import './Home.scss';
+import avatarImg from "../../Assets/6hk992di5pn61.jpg";
```

Comment:
I can see that you have used image: 6hk992di5pn61.jpg for preview. You can use placeholder image links from [Picsum](https://picsum.photos/) or [Placeholder](https://placeholder.com/) or any other image placeholder sites.
Remove the asset as this was not required and replace it with any placeholder image link for preview.